src/server/views/collection.hbs

```diff
@@ -2,4 +2,6 @@
 
 {{#contentFor "head"}}
 <title>Collection</title>
+<script>window._initialState = {{{initialState}}};</script>
```

Comment:
We have two choices, bootstrap the initial state to page load, or make an ajax call to get the initial state. Ideally, if we were to make an ajax call, it should be at the same time we are loading the enhanced version's scripts. Went with bootstrapping instead for a few reasons:

  - if the initial state is so large that it slows down page load, the problem sounds like it is in the design (we probably shouldn't be making it possible for people to be editing *that* much state on a page), and rendering such a large page will likely be a bigger issue than parsing the data
  - users on browsers getting the enhanced version are likely to need to wait longer to be able to do anything if we are only getting the state later via an ajax call.
  - if we want to load the initial state at the same time as the enhanced script, we need to do quite a bit of juggling to handle only start the enhancements once both the state has been loaded and the enhanced version's scripts have been loaded. Not impossible, but it does take some boilerplate, and I'm not sure if the returns are great enough.
  - things can be made simpler by first loading the enhanced version, then making an ajax request for the state it needs, but that is quite a bit of waiting
  - the bootstrapping approach is simpler, so seemed worth adding first so we can discuss it before trying anything else

I'm not completely against going the ajax-for-initial-state route though, both sound like they have their caveats.